### PR TITLE
P0 screens

### PR DESCRIPTION
**Pull request by rdshinde**
Added navbar for landing page.
Also added root color variables.

Screenshot 2022-02-14 at 9 56 17 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <div class="content">
+        <header class="header-container p-sm shadow-md">
+          <nav class="header__nav">
+            <div class="nav__logo-container">
+              <a href="./index.html" class="text-default"
+                ><i class="fas fa-store text-1"></i>&nbsp;
+                <h1 class="">
+                  Swadeshi
+                  <h4 class="bold-lg">.com</h4>
+                </h1>
```
Comment (ttahm3d) [inline index.html]: Dont use one heading tag inside another tag, Use span to do these kind of things

File: index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              </a>
+            </div>
+            <ul class="nav__links m-x-md">
+              <li class="m-x-sm p-x-sm">
+                <a class="btn btn-link p-b-lg" href="">Men</a>
```
Comment (ttahm3d) [inline index.html]: Dont leave empty `gref` in `a` tag. Use `#` if you dont have any thing to reference to.

**Review by ttahm3d — COMMENTED** [review]
Left few comments please take a look.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+/* Stling for nav__logo text */
+a h1,
+a h4 {
+  display: inline-block;
+}
+a .fa-store:hover {
+  transform: scale(1);
+}
```
Comment (theshivamverma) [inline style.css]: Use CSS styles to format the UI, instead of using random tags together.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            
+          </main>
+        </div>
+      </div>
+    </div>
+
+    <script
+      src="https://kit.fontawesome.com/fde0352739.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (theshivamverma) [inline index.html]: Use cdn in head to load external libraries.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+.content {
+  margin: auto;
+  position: relative;
+  width: 100%;
+}
+/* header comonent styling */
+.header-container {
+  position: sticky;
+  top: 0;
+  z-index: 999;
```
Comment (theshivamverma) [inline style.css]: can store z-indexes in variables and use, avoid random large numbers.

**Review by theshivamverma — CHANGES_REQUESTED** [review]

**Review by ttahm3d — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (rdshinde) [reply style.css]: Refactored this code and fixed all semantic error

File: index.html (hunk 4 above)
Comment (rdshinde) [reply index.html]: Added CDN to head tag.

File: style.css (hunk 5 above)
Comment (rdshinde) [reply style.css]: Created Variables for z-index.

**Pull request merged**